ERROR: AssertionError on line 5 ->
assert (
    x == 1 and y == 2 and z == 4
)
[0;31mERROR: `./testmain` exited with exitcode: 1[0m

exitcode=1